tasks/add_new_openwrap_partner.py

```diff
@@ -439,14 +440,15 @@ def setup_partner(user_email, advertiser_name, advertiser_type, order_name, plac
 
   #get device capabilty ids for in-APP platform
   device_capability_ids = None
-  if device_capabilities != None and creative_type is constant.IN_APP:
-      device_capability_ids = []
-      if isinstance(device_capabilities, str):
+  if device_capabilities != None:
+    if creative_type is constant.IN_APP or creative_type is constant.IN_APP_VIDEO:
```

Comment:
Instead of multiple `OR` conditions, we can use `IN` operator:
`if device_capabilities != None and creative_type IN (constant.IN_APP, constant.IN_APP_VIDEO)`